### ISSUE TITLE
[New Feature]: 

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-cs#195]
As a necessary prerequisite to getting https://github.com/unity-sds/unity-project-management/issues/71 done, we want to make sure some basic domain name support is in place on the Unity-Test account.

For this deployment, there will be two API Gateways:
1.  Public API Gateway in Unity-Test account
2. Private (or perhaps still publicly accessible?) in the Unity-Sips-Test account

In the Unity-Sips-Test account, project deployments will occur (e.g. SPS deployments), and these SPS endpoints need to be automatically hooked up to the API Gateways.   This ticket is about making the necessary "upserts" into the API Gateway routes, so that SPS will be hooked up properly upon deployment.

The types of routes that need to be inserted are:
- Service-level-routes:  routes between SPS endpoints (e.g. Mozart to Tosca), in the private API Gateway
- Shared Services routes:  route, for example that maps https://unity-sds.jpl.nasa.gov/sips/test to the private API Gateway
- Shared services routes from project account:  for example, shared service API Gateway that SPS uses to access the Data Catalog or DAPA API (see also: https://github.com/unity-sds/unity-cs/issues/192).  For these routes, @ramesh-maddegoda will most likely have to consult with @LucaCinquini or @drewm-jpl or @ryanhunter-jpl  for further information.

Acceptance Criteria:
- [ ] the above routes are automatically created as part of the Unity-Sips-Test project deployment

**Comment by LucaCinquini** [comment]
We should have a tag-up on this. If we are supporting multiple deployments in each venue, I would expect this kind of routing for the SIPS TEST venue and an EKS cluster named "fwd-processing":

WPS-T API endpoint: https://unity-sds.jpl.nasa.gov/sips/test/fwd-processing/wpst --> http://a0442fd7f829f49059fd68d1236aa263-650360946.us-west-2.elb.amazonaws.com:5001/

SPS API endpoint: https://unity-sds.jpl.nasa.gov/sips/test/fwd-processing/sps --> http://a7096fc6842e84da688b45586d194498-2116025617.us-west-2.elb.amazonaws.com:5002/

and similar for ADES specific endpoints if we want to support them, like the HySDS figaro and tosca.

**Comment by galenatjpl** [comment]
Pushing this ticket to 23.3 release.  We are partially blocked by MCP fully getting roles as code lambda integrated into their system.  Also, the fully automatic API registration we decided isn't ready for 23.2.

**Comment by galenatjpl** [comment]
This work is all basically done, except for the remaining work pertaining to the locking down at the network level of the access to gateways/subnets/components, etc..  Will create a separate ticket for that work.
